roles/keycloak/defaults/main.yml

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2023 VEXXHOST, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+keycloak_helm_release_name: keycloak
+keycloak_helm_chart_path: "../../charts/keycloak/"
+keycloak_helm_chart_ref: /usr/local/src/keycloak
+
+keycloak_helm_release_namespace: openstack
```

Comment:
let's maybe use something like `auth-system` in here? since we use keycloak in several different components/places like grafana (monitoring ns), openstack (openstakc ns), etc